frontend/pkg/frontend/helpers.go

```diff
@@ -219,6 +219,22 @@ func (f *Frontend) MarshalResource(ctx context.Context, resourceID *azcorearm.Re
 
 	logger := LoggerFromContext(ctx)
 
+	if resourceID.ResourceType.Type == (api.ClusterVersionTypeName) {
```

Comment:
This comparison should be case-insensitive.
```suggestion
	if strings.EqualFold(resourceID.ResourceType.Type, api.ClusterVersionTypeName) {
```